### ISSUE TITLE
Error using LMDB_ MDB_MAP_FULL: Environment mapsize limit reached

### ISSUE THREAD

**Issue by rfsantacruz** [kyamagu/matlab-lmdb#5]
I was trying to create a LMDB dataset with a huge MAPSIZE and when i insert a datum on it, it fires the following error: Error using LMDB_MDB_MAP_FULL: Environment mapsize limit reached.

What is the MAPSIZE maximum value ? 
And is there some work around to overcome this limitation ?

**Comment by kyamagu** [comment]
@rfsantacruz Not really, I usually specify a large value, say 1TB = `1024^4` or more. This is the design of LMDB.
